src/2024h2/parallel-front-end.md

```diff
@@ -0,0 +1,65 @@
+# Stabilize Parallel Front End
+
+| Metadata |                           |
+| -------- |---------------------------|
+| Owner(s) | [@SparrowLii]             |
+| Teams    | [Parallel Rustc WG] |
+| Status   | Existing problems need to be resolved to stabilize                          |
+
+## Motivation
+
+The parallel front end has been implemented in nightly, but there are still many problems that prevent it from being stable and used at scale.
```

Comment:
Sorry, I need to rework the tempalte to make this clearer, but can you add a summary that hits on "what you will do in next 6 months and what benefit to expect"?

Something like

> We will move rustc's support for parallel compilation closer to stability by resolving the XX known issues with rustc's parallel front-end and integrating parallel compilation into bootstrap. This fits into our larger goal of improving build times by N% by leveraging multiple cores.

Reply:
I updated the template etc -- see #31

Reply:
Thanks! I gonna update this